src/components/languageMenu/language-menu.less

```diff
@@ -0,0 +1,65 @@
+.languages-menu {
+  position: relative;
+  font-size: 10px;
+  color: #333;
+
+  .languages-menu__options {
+    position: absolute;
+    top: 3em;
+    right: 0;
+    display: none;
+    border-radius: 3px;
+    background-color: #fff;
+    box-shadow: 0 3px 6px rgba(0, 0, 0, 0.1), 0 3px 6px rgba(0, 0, 0, 0.13);
+    &_hidden {
```

Comment:
If I'm not wrong, according to BEM it should be "&--hidden"

Reply:
done

Reply:
"-" -  A hyphen is used to separate words in names
"_" - The value of the modifier is separated from the name of the modifier by one underscore